pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,7 +127,7 @@ public void testBackLogout() {
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
             .setRequestMethod("POST");
         TestsHelper.expectException(() -> casClient.getCredentials(context), HttpAction.class, "back logout request: no credential returned");
-        assertEquals(200, context.getResponseStatus());
+        assertEquals(204, context.getResponseStatus());
```

Comment:
I think it will regarding: https://github.com/apereo/cas/blob/master/core/cas-server-core-util/src/main/java/org/apereo/cas/util/http/SimpleHttpClientFactoryBean.java#L74, the `HTTP_NO_CONTENT` code is not in the list of the acceptable ones.